publish-wrapper

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2019 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import argparse
+import os
+import socket
+import subprocess
+import sys
+
+from task import github
+from task import sink
+
+sys.dont_write_bytecode = True
+
+HOSTNAME = socket.gethostname().split(".")[0]
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Log and publish into github')
+    parser.add_argument('--publish', help='Publish results centrally to a sink',
+                        default=os.environ.get("TEST_PUBLISH"))
+    parser.add_argument('--test-name', help='Test name',
+                        default=os.environ.get("TEST_NAME", "tests"))
+    parser.add_argument('--test-project', help="Project which is being tested",
+                        default=os.environ.get("TEST_PROJECT", ""))
+    parser.add_argument('--base-project', help="Project in which the tested PR is opened",
+                        default=os.environ.get("BASE_PROJECT"))
+    parser.add_argument('--test-branch', help="Branch to be tested - if differs to PR base branch",
```

Comment:
When would that be used? For our foreign tests, like testing starter-kit in a bots PR? Wouldn't that just be TEST_PROJECT=cockpit-starter-kit?

Reply:
If there is PR in `cockpit-project/bots` (like image refresh) and you test `context@cockpit-project/cockpit-podman` then:
- `TEST_PROJECT=cockpit-project/cockpit-podman` - used to create correct github context
- `BASE_PROJECT=cockpit-project/bots` - used so we know where to publish those contexts

Reply:
Since we are passing full github context, this indeed is not needed anymore.